zerver/openapi/openapi.py

```diff
@@ -15,6 +15,11 @@
         'post': {
             '200': ['max_message_id', 'realm_emoji']
         }
+    },
+    '/messages/{message_id}': {
+        'patch': {
+            '400': ['code']
+        }
```

Comment:
Why do we need this exception to be added change?

Reply:
I think the actual bug here is this should be marked in `zulip.yaml`  as  a `CodedError`?